src/AboutPage/About.tsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function About() {
   return (
-    <div>About</div>
+    <body>
+      <section className='flex justify-center items-center h-screen w-screen flex-col'>
+        <h1 className=' text-5xl m-10'>About</h1>
+        <img src="/src/images/hero-img.png" className='w-5/6 h 3/6'></img>
```

Comment:
Not sure what the `h 3/6` is meant for, but it's generally bad practice to set height anyways since it should adjust depending on the width as the screen resizes.